innovation_sweet_spots/getters/inputs.py

```diff
@@ -10,6 +10,7 @@
 import pandas as pd
 from pandas import read_sql_table
 import yaml
+import urllib.request
 
 INPUTS_PATH = f"{PROJECT_DIR}/inputs/data/"
 GTR_PATH = f"{INPUTS_PATH}gtr_projects.csv"
```

Comment:
If `PROJECT_DIR` was a `pathlib.Path` object, then you could change these to:

```suggestion
INPUTS_PATH = PROJECT_DIR / "inputs/data/"
GTR_PATH = INPUTS_PATH / "gtr_projects.csv"
```